### PR TITLE
Add test for asi between `get`/`set` field and generator

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
Currently both SpiderMonkey and TypeScript fail parsing this code. V8, Babel and ESLint all pass the test.

### CODE REVIEW

File: test/language/statements/class/elements/syntax/valid/grammar-field-named-get-followed-by-generator-asi.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2024 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: ASI between a field named "get" and a generator method
+esid: prod-ClassElement
+features: [class-fields-public, class]
```
Comment (linusg) [inline test/language/statements/class/elements/syntax/valid/grammar-field-named-get-followed-by-generator-asi.js:6]:
```suggestion
features: [class-fields-public, class-static-fields-public, class, generators]
```

**Comment by JLHwung** [comment]
Could you also add a set ASI test?

**Comment by nicolo-ribaudo** [comment]
Test case with `set` added

**Comment by nicolo-ribaudo** [comment]
cc @anba for a review since you fixed this in SM :)

File: test/language/statements/class/elements/syntax/valid/grammar-field-named-set-followed-by-generator-asi.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2024 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: ASI between a field named "get" and a generator method
```
Comment (anba) [inline test/language/statements/class/elements/syntax/valid/grammar-field-named-set-followed-by-generator-asi.js:4]:
```suggestion
description: ASI between a field named "set" and a generator method
```

**Review by anba — APPROVED** [review]
LGTM with the copy-paste error fixed.

**Pull request merged**